documents/src/pages/elements/combo-box.md

```diff
@@ -46,7 +46,8 @@ comboBox.data = [
 `ef-combo-box` displays a text input and an associated pop-up element that helps users set a value.
 
 ## Usage
-The `ef-combo-box` uses the `data` property that follow [ComboBoxData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/combo-box/helpers/types.ts) interface.
+
+The `data` property of the `ef-combo-box` use the [ComboBoxData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/combo-box/helpers/types.ts) interface for its data items. Each item defaults to type ItemData, it can be anything extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
```suggestion
The `data` property of the `ef-combo-box` use the [ComboBoxData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/combo-box/helpers/types.ts) interface for its data items. Each item defaults to type ItemData, otherwise it can be any interface that extended from [DataItem](./custom-components/utils/data-management#data-item).
```